pom.xml

```diff
@@ -104,6 +104,18 @@
       <version>${stack.version}</version>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-rx-java3</artifactId>
+      <version>${stack.version}</version> <!-- FIXME remove once it is in BOM -->
+      <optional>true</optional>
+    </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-rx-java3-gen</artifactId>
+      <version>${stack.version}</version> <!-- FIXME remove once it is in BOM -->
```

Comment:
@jo5ef currently its not included in vertx-dependencies... hopefully they do that before release